xbmc/utils/EGLUtils.cpp

```diff
@@ -667,13 +669,15 @@ void CEGLContextUtils::SetDamagedRegions(const CDirtyRegionList& dirtyRegions)
   }
   else
   {
-    EGLint height = eglQuerySurface(m_eglDisplay, m_eglSurface, EGL_HEIGHT, &height);
```

Comment:
https://registry.khronos.org/EGL/sdk/docs/man/html/eglQuerySurface.xhtml

EGLInt height = 0;
EGLBoolean ret = eglQuerySurface(m_eglDisplay, m_eglSurface, EGL_HEIGHT, &height);

Should return true and height should now be updated properly? The original syntax was not correct.

Reply:
Yeah.'m not sure what I was thinking when writing that.